odoo/addons/base/static/tests/base_settings_tests.js

```diff
@@ -83,6 +83,8 @@ QUnit.module('base_settings_tests', {
         assert.strictEqual($('.highlighter').html(),"B","b word hilited");
         form.$('.searchInput').val('bx').trigger('keyup');
         assert.strictEqual(form.$('.notFound').hasClass('o_hidden'),false,"record not found message shown");
+        form.$('.searchInput').val('f').trigger('keyup');
+        assert.strictEqual($('.highlighter:eq(2)').html(),"F","F word hilited");
```

Comment:
- only look inside form.$el -> form.$('.highlighter');
- why do you need eq(2)?
- hilited -> highlighted

Reply:
Typo `hilited` is already being used at other places.
Maybe we can Improve those too
![Screenshot from 2019-05-21 00-55-29](https://user-images.githubusercontent.com/16624719/58046521-31438c80-7b63-11e9-9ec1-69a3faca9772.png)

Reply:
@aab-odoo changes are updated. Can you please check now?